README.md

```diff
@@ -1 +1,47 @@
-# Review_Trials_Project
\ No newline at end of file
+# Review Trial Project
```

Comment:
Great job for fixing the miswriting that was preventing you from having a professional repository 👍

Reply:
Good job using polite and clear language as well as emojis to make the comment human-friendly.